From: Vincent Lefevre <vincent@vinc17.net>
Date: Fri, 16 Dec 2022 18:20:19 +0100
Subject: builtin: Actually accept ulimit -r

The original commit that added it supposes this works, but it only adds
it to the ulimit -a listing and the manual, but doesn't allow it as an
option.

Fixes: 46abc8c6d8a5 ("[BUILTIN] Add support for ulimit -r")
Bug-Debian: https://bugs.debian.org/975326
---
 src/miscbltin.c | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/src/miscbltin.c b/src/miscbltin.c
index 5ccbbcb..e553f9e 100644
--- a/src/miscbltin.c
+++ b/src/miscbltin.c
@@ -440,6 +440,9 @@ ulimitcmd(int argc, char **argv)
 #endif
 #ifdef RLIMIT_LOCKS
 			       "w"
+#endif
+#ifdef RLIMIT_RTPRIO
+			       "r"
 #endif
 	)) != '\0')
 		switch (optc) {
